docs/index.md

```diff
@@ -11,74 +11,47 @@ layout: default
 </div>
 </header>
 
-<section>
+<section id="section-snippet">
 <div class="wrapper">
 <h1 class="title">
-About
+Easily Create Fast, Lightweight Web Components
```

Comment:
+1. This is an important distinction from the previous wording... it's about creating new custom components, not a collection of existing components that are fast and lightweight.